app/nav/Navbar.tsx

```diff
@@ -42,9 +42,9 @@ export default memo(function Navbar({
 	);
 
 	return (
-		<chakra.nav borderBottomWidth="1px" w="full" pos="sticky">
-			<Flex w="full" maxW="7xl" mx="auto" align="center">
-				<Box p={2} px={4}>
+		<chakra.nav borderBottomWidth="1px" w="full" pos="sticky" p="0 1em">
```

Comment:
- Padding shouldn't be a string value, but numeric (`px`, `py`)
- Why not apply the padding to the first element (the heading box) as before?